components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -4306,7 +4306,7 @@ document.observe("xwiki:dom:loaded", function() {
     },
 
     getElement : function() {
-      return this.span;
+      return this.span.down();
     },
```

Comment:
This will break other things, since now instead of inserting <span> the content of <span> will be inserted into the document. And <span> is there for a reason, I think IE9 (which som ehoospitals still use) is broken without it (see comment a few lines above).

Instead, if <span? is the problem, you can exclude <span> from the tab order, e.g. set tabindex="-1" or something like that
(http://stackoverflow.com/questions/3895710/explicitly-exclude-an-html-element-from-the-tab-order, https://www.google.ca/search?q=html+exclude+form+tab+order&oq=html+exclude+form+tab+order)